db/seeds/posts/tos.html

```diff
@@ -14,17 +15,20 @@ <h2 id="2-member-conduct">2. Member Conduct</h2>
     <li>post content which you do not have the right or permission to post, or to infringe any copyright or other proprietary right of any entity or person;</li>
     <li>post content that is hateful, discriminatory, harmful, illegal, or otherwise objectionable;</li>
     <li>impersonate any other person, group, or entity;</li>
-    <li>violate the then-current Code of Conduct of the Service, which is available <a href="https://writing.codidact.com/help/code-of-conduct">in our help center</a>.</li>
+    <li>violate the then-current Code of Conduct of the Service, which is available in our help center. <!-- $EDIT-add-link --> </li>
```

Comment:
Minor: You could link to `/help/code-of-conduct`, which will link to the help article on the current site.